datamol/mol.py

```diff
@@ -1376,3 +1384,42 @@ def get_atom_positions(
         positions = positions[mapped_indices, :]
 
     return positions
+
+
+def remove_salts(mol: Mol, remover: Optional[SaltRemover] = SALT_REMOVER) -> Mol:
+    """Remove all salts from the molecule
+
+    Args:
+        mol: A molecule.
+        remover: A object that defines salts to be removed.
+
+    See Also:
+        <rdkit.Chem.SaltRemover.SaltRemover>
```

Comment:
Can you add link to the upstream doc in the docstring: https://www.rdkit.org/docs/source/rdkit.Chem.SaltRemover.html

It's something I am trying to do more often as I found it to be quite convenient when working with datamol myself.

Reply:
Actually forgot about that. I don't think it's relevant for that specific function.